docs/usage/app-icon.md

```diff
@@ -0,0 +1,7 @@
+# App Icon
+
+The App Icon should allow a user to quickly and uniquely identify your application. It will appear in the DHIS2 App Hub and in the list of available applications in the Headerbar of a running DHIS2 instance.
+
+## Specifying a Custom App Icon
+
+To specify a custom App Icon, place the relevant file in the `public` directory and name it `dhis2-app-icon.png`. It must be in PNG format, and should include a transparent background. The recommended size of this image is 48x48 pixels. When no `public/dhis2-app-icon.png` is provided, a default icon (the DHIS2 logo) will be used.
```

Comment:
I'd remove this: "(the DHIS2 logo)" so we can just change the default icon without having to remember to update the docs.

Reply:
Will do